modules/elb/route-53.tf

```diff
@@ -0,0 +1,11 @@
+data "aws_route53_zone" "hosted_zone" {
+  name = var.domain_name
+}
+
+resource "aws_route53_record" "devops_example" {
+  zone_id = data.aws_route53_zone.hosted_zone.zone_id
+  name    = var.record_name
+  type    = "CNAME"
+  ttl     = 300
+  records = [aws_lb.public-alb]
```

Comment:
this should be `records = [aws_lb.public-alb.dns_name]` could you please fix it.